.github/wiki/Fastlane.md

```diff
@@ -1,150 +1,146 @@
 
 > See more:
 >
 > - [Build and upload the application](#build-and-upload-the-application)
 
-### DistributionManager.rb
+### Distribution.swift
 
-The `DistributionManager` is in charge of distributing the build to the distribution tools, such as Firebase, App Store Connect, and Testflight.
+The `Distribution` is in charge of distributing the build to the distribution tools, such as Firebase, App Store Connect, and Testflight.
 
 > See more:
 >
 > - [Build and upload the application](#build-and-upload-the-application)
 
-### MatchManager.rb
+### Match.swift
 
-The responsibility of `MatchManager` is to synchronize the certificates and profiles across the team during the development process.
+The responsibility of `Match` is to synchronize the teams' certificates and profiles during the development process.
 
 > See more:
 >
 > - [Synchronize the provisioning profile](#synchronize-the-provisioning-profile)
 > - [Matchfile](#matchfile)
 
-### SymbolManager.rb
+### Symbol.swift
 
-Technically, the debug Symbol file (dSYM file) is used to de-obfuscate stack traces from crashes happening on the production app. The dSYM files store the debug symbols for the application. Then services (like Crashlytics) use the dSYM files to replace the symbols in the crash reports with the originating locations in the source code. Hence, the crash reports will be more readable and understandable.
+Technically, the debug Symbol file (dSYM file) is used to de-obfuscate stack traces from crashes on the production app. The dSYM files store the debug symbols for the application. Then services (like Crashlytics) use the dSYM files to replace the symbols in the crash reports with the originating locations in the source code. Hence, the crash reports will be more readable and understandable.
 
-The `SymbolManager` helps process and upload dSYM file to Firebase. There are two main functions:
+The `SymbolManager` helps process and upload dSYM file to Firebase. There is only one primary function:
 
-| `upload_built_symbol_to_firebase` | `download_processed_dsym_then_upload_to_firebase` |
-|---|---|
-| Directly upload the built dSYM file to Crashlytics. | Download the processed dSYM file from App Store Connect, then upload it to Crashlytics. |
-|  It is recommended to use this function when the build configuration is `Release Staging`. | It is recommended to use this function when the build configuration is `Release Production`. |
-| See more: [Debug Symbol File](Project-Configurations.md#debug-symbol-file) | See more: [Debug Symbol File](Project-Configurations.md#debug-symbol-file) and [Enable Bitcode](Project-Configurations.md#enable-bitcode) |
+| `uploadToCrashlytics` |
+|---|
+| Directly upload the built dSYM file to Crashlytics with the environment parameter  (`staging` or `production`). |
```

Comment:
PoW looks correct, but in this `.md`, I think maybe we have one extra space. Please help me to recheck it. Thank you 🙏 .
```suggestion
| Directly upload the built dSYM file to Crashlytics with the environment parameter (`staging` or `production`). |
```